pkg/virt-launcher/virtwrap/converter/archconverter.go

```diff
@@ -59,3 +61,34 @@ func defaultSCSIController(c *ConverterContext, driver *api.ControllerDriver) ap
 		Driver: driver,
 	}
 }
+
+func defaultDomainVCPUTopologyForHotplug(vmi *v1.VirtualMachineInstance, domain *api.Domain) {
```

Comment:
I agree, the alternative is to declare the same logic in all architectures as otherwise the code simply won't compile.

As for the your suggestion for something along the lines of `hasCPUHotPlug()`, I don't mind doing that, but it just delays moving the function to when ARM64 eventually supports hotplugging.

Reply:
I'll do that as right now it does seem to be the elegant solution.

Reply:
> the alternative is to declare the same logic in all architectures as otherwise the code simply won't compile.

I think we compile all files on all arches, as the architecture here is that of the guest, which may differ from the architecture that runs the controllers (though not the case of virt-launcher).